app/code/Magento/Ui/view/base/web/js/lib/core/element/links.js

```diff
@@ -79,6 +79,10 @@ define([
 
         component.set(property, value, owner);
 
+        if (property === 'disabled' && value) {
```

Comment:
I believe `updateValue` shouldn't change the `validate` property when an element receives the 'disabled' property, that may introduce undesirable behaviour to other components. Instead, I would recommend a change more towards the component itself recognizes that is being disabled and update the validation.

As an example, `app/code/Magento/Ui/view/base/web/js/form/element/abstract.js` has the validation method that considers if the element is disabled, so basically we need to validate the component again if the component is disabled, that will undo the validation issue.